safetrace-sdk/SafeTrace.swift

```diff
@@ -84,6 +84,7 @@ public final class SafeTrace {
     public static func applicationWillEnterForeground(_ application: UIApplication) {
         environment.traceIDs.refreshIfNeeded()
         environment.tracer.reportPendingTraces()
+        environment.session.updateAuthTokenWebViewCookies(authToken: environment.session.authToken)
```

Comment:
seems a bit weird to pass the token in here when the session already has access to it, but ok

Reply:
Yeah I thought the same. I can change the public interface to not take in an authToken parameter, and have that handled internally in UserSession